templates/csv.yaml

```diff
@@ -125,47 +125,36 @@ spec:
     This operator provides the following components:
 
     ## Knative Serving
-    Knative Serving builds on Kubernetes to support deploying and serving of
-    applications and functions as serverless containers. Serving is easy to get
-    started with and scales to support advanced scenarios. Other features
-    includes:
-    - Rapid deployment of serverless containers
-    - Automatic scaling up and down to zero
+    Knative Serving builds on Kubernetes to support deploying and serving of applications and functions as serverless containers.
+    Serving simplifies the application deployment, dynamically scales based on in incoming traffic and supports custom rollout strategies with traffic splitting.
+    Other features include:
+    - Simplified deployment of serverless containers
+    - Traffic-based auto-scaling, including scale-to-zero
     - Routing and network programming
-    - Point-in-time snapshots of deployed code and configurations
-    ![](https://i.imgur.com/vqL48B8.png)
+    - Point-in-time application snapshots and their configurations
 
     ## Knative Eventing
-
-    Knative Eventing is a system that is designed to address a common need for cloud native
-    development and provides composable primitives to enable late-binding event sources and
+    Knative Eventing provides a platform that offers composable primitives to enable late-binding event sources and
     event consumers.
-    Knative Eventing is designed to address a common need for cloud
-    native development:
-    - Services are loosely coupled during development and deployed independently
-    - A producer can generate events before a consumer is listening, and a consumer
-       can express an interest in an event or class of events that is not yet being
-       produced.
+    Knative Eventing supports the following architectural cloud-native concepts:
+
+    - Services are loosely coupled during development and deployed independently to production
+    - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications
        * without modifying producer or consumer, and
        * with the ability to select a specific subset of events from a particular
-         producer.
```

Comment:
Did you intend to delete this word? The preceding bullet doesn't make sense without it.

Reply:
good catch. Indeed a c&p error.